src/App.js

```diff
@@ -4,22 +4,22 @@ import "./App.css";
 import Header from "./components/Header";
 import Add from "./components/pages/Add";
 import Vote from "./components/pages/Vote";
-import Diy from "./components/pages/Diy";
+import Result from "./components/pages/Result";
 
 function App() {
   return (
     <Router>
       <Header />
-      <main>
+      <main className="main">
```

Comment:
since you'll only have one `main` element you don't really need the `className` here, unless you have a system that uses/declares several layout classes or a very specific one